KUBERNETES_CSI_OWNERS_ALIASES

```diff
@@ -4,7 +4,7 @@ aliases:
 
   # SIG-Storage chairs and leads should always have approval rights in all repos.
   # Others may be added as needed here or in each repo.
-  kubernetes-csi-approvers:
+  approvers:
```

Comment:
you can leave the `kubernetes-csi-` here. this is an alias.

Reply:
done